base/stats.go

```diff
@@ -986,6 +986,11 @@ type SgwIntStat struct {
 	AtomicInt
 }
 
+type SgwUIntStat struct {
```

Comment:
very much a nit:

I get that this name is trying to match `AtomicUInt` but I'd prefer the SgwUintStat or SgwUint64Stat to match atomic.Uint64